### PR TITLE
Bump version to 0.92.0

### PR DESCRIPTION
**Pull request by bwoebi**
++$version;

### CODE REVIEW

**Review by Anilm3 — DISMISSED** [review]
lgtm

**Comment by morrisonlevi** [comment]
[System tests has a repeated failure in there](https://app.circleci.com/pipelines/github/DataDog/dd-trace-php/12868/workflows/ef040b22-49a7-4b51-b6d3-0d503f8f00fb/jobs/2930965):

> ValueError: Schema is invalid in logs/interfaces/agent/00008__api_v2_apmtelemetry.json

I didn't investigate further, I am about to go to a doctor appointment.

File: components-rs/common.h [hunk 1]
```diff
@@ -72,7 +72,13 @@ typedef struct ddog_Error {
  * so every operation on it is unsafe!
  */
 typedef struct ddog_Slice_CChar {
+  /**
+   * Must be non-null and suitably aligned for the underlying type.
+   */
   const char *ptr;
+  /**
+   * The number of elements (not bytes) that `.ptr` points to.
```
Comment (morrisonlevi) [inline components-rs/common.h:80]: Maybe in libdatadog I should modify this to say:

> The number of elements (not bytes, unless they are the same for a given type) that `.ptr` points to.

Seems misleading for `const char` ^_^

**Review by Anilm3 — APPROVED** [review]
The system tests now pass locally with https://github.com/DataDog/system-tests/pull/1624

**Pull request merged**